server/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+/dist
```

Comment:
By the way, I just noticed it doesn't look like you synced these to `.dockerignore` (I only noticed in your next PR): could you add them there? Might be nice to add a comment to this `.gitignore` reminding you to sync to `.dockerignore` too.